app/assets/main/react_components/footprint_form/FootprintForm.jsx

```diff
@@ -198,16 +190,34 @@ const FootprintForm = ({ calculator, questions, options, footprint }) => {
     return ""
   }
 
+  /**
+   * Called when the answer to a question is given, saves the result and loads the next question
+   */
+  function onAnswerGiven(givenAnswer){
+    saveAnswer(givenAnswer);
+    increaseIndex();
+    if(questionIndex == -1){
+      submit();
+    } else {
+      if (questionKeys[questionIndex-1] === "car_type" && givenAnswer === "no_car" ){
+        saveAnswer("");
+        increaseIndex();
+        delete questionCategories["car_distance"]      }
+      setQuestion();
+    }
+  }
+
   return (
       <form action="/calculator" acceptCharset="UTF-8" method="post" onSubmit={e => { e.preventDefault(); }}>
         <div className="question py-8" data-target="lifestyle-footprints--calculator.question" data-category="home">
+          <ProgressBar questionCategories={questionCategories} calculator={calculator} activeCategory={category} activeQuestion={questionKeys[questionIndex]}/>
```

Comment:
Kanske dela upp i två rader för läsbarhetens skull?

Reply:
Jag förstår inte frågan om region! Var kommer region in? Kollar jag på fel ställe? 
Delar upp i två!

Reply:
Denna var bara för för att det var en lång rad och tänkte att man kan dela upp den om man vill. Inget mer än så :)